lib/ansible/modules/cloud/azure/azure_rm_image.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
```

Comment:
There's a new header that is being used for Ansible modules, makes things a bit simpler than before.

```
#!/usr/bin/python
# Copyright: ...
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Reply:
Done, in both modules